src/react/components/pages/projectSettings/projectForm.test.tsx

```diff
@@ -0,0 +1,223 @@
+import React from "react";
+import ProjectForm from "./projectForm";
+import { mount } from "enzyme";
+import { MockFactory } from "../../../../models/mockFactory";
+import TagColors from "../../common/tagsInput/tagColors.json";
+
+import { Provider } from "react-redux";
+import { BrowserRouter as Router } from "react-router-dom";
+import { IApplicationState } from "../../../../models/applicationState";
+import initialState from "../../../../redux/store/initialState";
+import createReduxStore from "../../../../redux/store/store";
+import ProjectSettingsPage from "./projectSettingsPage";
+
+describe("Project Form Component", () => {
+    let onSubmit: (value: any) => void;
+
+    const mockFactory = new MockFactory();
+    const project = mockFactory.project();
+    const connections = mockFactory.connections();
+    const defaultState: IApplicationState = initialState;
+    const store = createReduxStore(defaultState);
+
+    describe("Existing project", () => {
+        onSubmit = jest.fn();
+        let wrapper: any = null;
+
+        beforeEach(() => {
+            wrapper = mount(
+                <Provider store={store}>
+                    <Router>
+                        <ProjectForm
+                            project={project}
+                            connections={connections}
+                            onSubmit={onSubmit}
+                            />
+                    </Router>
+                </Provider>,
+            ).find(ProjectForm).childAt(0);
+        });
+
+        it("should have name loaded correctly", () => {
```

Comment:
You could likely combine these into a single test called `Should initialize state correctly` and then have all of your expectations there.  The initialization of these tests are all the same and you are causing the tests to run more slowly having to reinstantiate the wrapper/component for each test.

Reply:
makes sense. I'll make that change